src/util/scripts.js

```diff
@@ -2,6 +2,9 @@ const path = require('path');
 const { spawn } = require('promisify-child-process');
 const Config = require('./config');
 
+/**
+ * @deprecated since v1.2.0
```

Comment:
related to previous menitoned `deprecated `

Reply:
Answered above